marklogic-data-hub-central/README.md

```diff
@@ -78,6 +78,10 @@ If you are connecting to a remote Data Hub instance, you can override mlHost to
 
     ./gradlew bootRun -PmlHost=somehost
 
+To associate the middle tier with a local Data Hub project, use the following:
+
+    ./gradlew bootRun -PhubProjectPath=/path/to/project
+
```

Comment:
I believe Rob's PR #3864 would remove dependency on local project. So, after that PR gets merged , this will not be true